src/randoop/ExceptionCheck.java

```diff
@@ -0,0 +1,145 @@
+package randoop;
+
+import java.io.ObjectStreamException;
+import java.lang.reflect.Modifier;
+import java.util.Objects;
+
+/**
+ * An {@code ExceptionCheck} is used to indicate that an exception is expected
+ * at a particular statement in a sequence. Depending on command line arguments
```

Comment:
Minor nit: "command-line" as an adjective contains a hyphen.  ("command line" as a noun does not.)